### PR TITLE
Present final project in main branch

### PR DESCRIPTION
**Pull request by SaveryIV**
PR for present the hole project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,198 @@
...
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
```
Comment (MahmoudBakr23) [inline README.md:13]: - [ ] I'm afraid your team forgot to record a maximum min presentation video of the project, please manage to do so as per project requirements and attach the link to the presentation in the README file. 🙏

File: classes/app.rb [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    print 'Label: '
+    game.label = gets.chomp
+    print 'Archived (true/false): '
+    game.archived = gets.chomp
+
+    @games << game
+    write_to_file(@games, './data/games.json')
+    # File.write('games.json', JSON.generate(@games))
+    puts 'New game added successfully'
+  end
```
Comment (MahmoudBakr23) [inline classes/app.rb:92]: - [ ] While creating a game, we are creating the author, label, and genre right? Although, the game is being saved correctly but its information doesn't get listed in their specified sections. For example, when I create a game with the author: 'Jhon', then I choose to list all authors, it shows me nothing! The same for the genres and labels. Please work on this issue. 🙏 
Welcome - App_My_Things_Capstone_Project - Visual Studio Code 10_27_2023 4_13_48 AM

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello TEAM 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done on creating the classes of the projects and the custom methods
- Nice work with the UI of the program
- Well done preserving the data on program exit
- Good job with the RSpec tests
- Descriptive PR and README file
- Correct Linters

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @SaveryIV @albertkantwi @AjroudiRida 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All tests passed 👍 
* App works well 👍 
* Professional video presentation 👍 
* No linter errors 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**